src/components/WorkflowForms/NewClientSignup/GatherInfo/i18n-strings.js

```
@@ -0,0 +1,188 @@
+    "reportNumberRequired": "Police report number is required when a police report exists",
+    "agencyRequired": "Agency is required when a police report exists",
+    
+    // PassengersSection
+    "passengersSectionTitle": "Additional Passengers",
+    "noPassengers": "No additional passengers added.",
+    "newPassenger": "New Passenger",
+    "name": "Name",
+    "phone": "Phone",
+    "savePassenger": "Save Passenger",
+    "cancel": "Cancel",
+    "fileUploadError": "Error uploading file",
```

ESLint

[error] Duplicate key 'fileUploadError'.

(no-dupe-keys)